src/KOKKOS/pair_mliap_kokkos.cpp

```diff
@@ -165,6 +176,8 @@ void PairMLIAPKokkos<DeviceType>::settings(int narg, char ** arg)
     } else
       iarg++;
   }
+  PairMLIAP::settings(narg, arg);
```

Comment:
I am a bit confused by seeing this statement here with unmodified arguments. I would have expected that any options that are parsed in this class would be removed from the list of arguments before passing it to the base class for further processing.

Reply:
This has been pushed.  @akohlmey if you are happy I think if you remove changes requested I can push once the tests pass.